api/filter/http_fault.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter;
+
+import "api/rds.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/wrappers.proto";
+
+// Delay specification is used to inject latency into the rpc operations.
+message Delay {
+  // Delay type to use (fixed|exponential|..). Currently, only fixed delay (step function) is supported.
+  string type = 1;
```

Comment:
Why not enum?

Reply:
I was going by what was defined in the configs (where type was a string). I guess Enum seems more sensible given we are doing protos.